source/core_json.c

```diff
@@ -51,6 +51,12 @@ typedef union
     ( ( ( x ) == ' ' ) || ( ( x ) == '\t' ) || \
       ( ( x ) == '\n' ) || ( ( x ) == '\r' ) )
 
+#define isOpenBracket_( x )     ( ( ( x ) == '{' ) || ( ( x ) == '[' ) )
+#define isCloseBracket_( x )    ( ( ( x ) == '}' ) || ( ( x ) == ']' ) )
+/* NB. The numeric values of the open and close bracket pairs differ by 2. */
```

Comment:
What does "NB" stand for?

Reply:
Nota bene

Reply:
May be _esoteric_ for readers?

Reply:
We use e.g. and i.e. often enough.  I'll keep it.